web-project/src/main/java/com/epam/webproject/controller/UserController.java

```diff
@@ -0,0 +1,24 @@
+package com.epam.webproject.controller;
+
+import com.epam.webproject.service.AuthorService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/user")
+public class UserController {
+
+    @Autowired
+    private AuthorService authorService;
+
+    @RequestMapping("/getAuthor")
```

Comment:
I suggest to use REST naming conventions for url path (resources) and http methods to describe the action.

REST naming patterns/antipatterns: http://www.restapitutorial.com/lessons/restfulresourcenaming.html
Must-read about http: https://habrahabr.ru/company/yandex/blog/265569/